src/ngraph/runtime/gpu/gpu_call_frame.cpp

```diff
@@ -77,7 +76,7 @@ void runtime::gpu::GPU_CallFrame::tensor_call(
         outputs.push_back(tv->m_allocated_buffer_pool);
     }
 
-    m_compiled_function(inputs.data(), outputs.data(), m_cublas_handle, m_cudnn_handle);
+    m_compiled_function(inputs.data(), outputs.data(), m_external_function->m_ctx.get());
```

Comment:
what if we also pass here as a reference to unique_ptr

Reply:
See my comment below regarding interoperability between the clang and native compiled code. For this reason we should use only raw pointers.

Reply:
OK. I see.